packages/rest-typings/src/v1/users.ts

```diff
@@ -367,6 +368,16 @@ export type UsersEndpoints = {
 	'/v1/users.deleteOwnAccount': {
 		POST: (params: { password: string; confirmRelinquish?: boolean }) => void;
 	};
+
+	'/v1/users.getNames': {
+		GET: (params: UsersGetNamesParamsGET) => {
+			users: {
+				userId: string;
+				username: string;
+				name: string;
+			}[];
```

Comment:
```suggestion
			users: Pick<IUser, '_id', 'username', 'name'>[];
```

Reply:
@matheusbsilva137 the fields are different tho 😬 also, i think, in IUser username is nullable, so is name, which again isn't what the endpoint is returning (the nullability isn't true, it is nullable for the sake of reusing the types in many places, does not signify username or name won't be in the resulting record (name prob for rocket.cat idk))